lib/Doctrine/DBAL/SQLParserUtils.php

```diff
@@ -196,16 +204,46 @@ static public function expandListParameters($query, $params, $types)
      * 0 => matched fragment string,
      * 1 => offset of fragment in $statement
      *
-     * @param string $statement
+     * @param string                $statement
+     * @param AbstractPlatform|null $platform The platform to use for unquoting.
+     *
      * @return array
      */
-    static private function getUnquotedStatementFragments($statement)
+    static private function getUnquotedStatementFragments($statement, AbstractPlatform $platform = null)
```

Comment:
Since it's a private method we can add the type declarations and use `?AbstractPlatform $platform` instead of `AbstractPlatform $platform = null`